contract/src/main/java/io/neow3j/contract/Nep5Token.java

```diff
@@ -6,6 +6,8 @@
 import io.neow3j.wallet.Account;
 import io.neow3j.wallet.Wallet;
 import io.neow3j.wallet.exceptions.InsufficientFundsException;
+import org.bouncycastle.util.Strings;
```

Comment:
this should not be here. :-)

Reply:
idk how that was created, I'll remove it :)